Swift/AEPSampleApp/EdgeIdentityView.swift

```diff
@@ -7,13 +7,31 @@
  it.
  */
 
-import Foundation
+import AEPCore
 import AEPEdgeIdentity
+import Foundation
 import SwiftUI
 
 struct EdgeIdentityView: View {
     @State var currentEcid = ""
     @State var currentIdentityMap: IdentityMap?
+    @State var adID: UUID?
+    @State var adIdText: String = ""
+    @State var trackingAuthorizationResultText: String = ""
+    
+    /// Updates view for ad ID related elements
+    func updateUsingTrackingAuthorization() {
```

Comment:
renaming suggestion: setDeviceAdvertisingIdentifier()

Reply:
Thank you! Refactored to suggested name